src/stylesheets/components/_accordions.scss

```diff
@@ -1,16 +1,12 @@
 // Variables
 
-$accordion-border: 3px solid $color-gray-lightest;
+$accordion-border: 3px solid color('gray-5');
 
 // Accordion Styles
 
-$icon-size: $spacing-md-small;
```

Comment:
You _could_ set this to `units(2.5)` and use the var as it was used previously

Reply:
And we _could_  build a new set of vars like those that start at https://github.com/uswds/uswds/blob/ca2b10fef9278a334715b6179e069a4a89e37a8e/src/stylesheets/core/_variables.scss#L1139

...and use _them_ in these var assignments. Like:
```sass
$theme-spacing-x-small: units(0.5); // was 0.5rem
$theme-spacing-small: units(1); // was 1rem
$theme-spacing-md-small: units(2); // was 1.5rem;
$theme-spacing-medium: units(2.5); // 2rem;
$theme-spacing-large: units(4); // 3rem;
```

```sass
$icon-size: $theme-spacing-md-small;
$h-padding: $theme-spacing-medium;	
$v-padding: $theme-spacing-md-small;
```

Reply:
Whoops, nm. It's goes against the paradigm a bit to use functions in settings. The more reliable thing would be more like:

```sass
$theme-spacing-x-small: 0.5; // was 0.5rem
$theme-spacing-small: 1; // was 1rem
$theme-spacing-md-small: 2; // was 1.5rem;
$theme-spacing-medium: 2.5; // 2rem;
$theme-spacing-large: 4; // 3rem;
```

```sass
$icon-size: units($theme-spacing-md-small);
$h-padding: units($theme-spacing-medium);	
$v-padding: units($theme-spacing-md-small);
```

Reply:
I think I'm not a huge fan of that? Like, we have all of the tokens, and this is an internal component, so why add the additional unnecessary abstraction of a new variable for this component? 

Unless we add them to variables that get used throughout the components?

Reply:
actually, forget this! I totally misread what you wrote, adding those spacing vars to the variables file seems like a good idea!

Reply:
I was just going off of what @thisisdano wrote above. If there is a different standard I'll switch to that.